src/Components/IndexScene/IndexScene.tsx

```diff
@@ -304,6 +320,60 @@ export class IndexScene extends SceneObjectBase<IndexSceneState> {
     return getContentScene(this.state.routeMatch?.params.breakdownLabel);
   }
 
+  private provideAssistantContext() {
+    const setAssistantContext = providePageContext(`${PLUGIN_BASE_URL}/**`, []);
```

Comment:
The providePageContext() seems like a global event emitter which might prove hard to maintain.